engines/sci/graphics/view.cpp

```diff
@@ -820,12 +795,12 @@ void GfxView::draw(const Common::Rect &rect, const Common::Rect &clipRect, const
 	const int16 width = MIN(clipRect.width(), celWidth);
 	const int16 height = MIN(clipRect.height(), celHeight);
 
-	bitmap += (clipRect.top - rect.top) * celWidth + (clipRect.left - rect.left);
+	const byte *bitmapData = bitmap.getUnsafeDataAt((clipRect.top - rect.top) * celWidth + (clipRect.left - rect.left), width * height);
```

Comment:
This `width * height` doesn't seem correct for (horizontally) clipped views.

I'm thinking `celWidth * height - (clipRect.left - rect.left)` ?